osf/utils/machines.py

```diff
@@ -173,7 +173,7 @@ def notify_submit(self, ev):
         context = self.get_context()
         context['contributors_url'] = '{}contributors/'.format(self.machineable.target.absolute_url)
         context['project_settings_url'] = '{}settings/'.format(self.machineable.target.absolute_url)
-        for admin in self.machineable.target.admin_contributors:
+        for admin in self.machineable.target.contributors.filter(contributor__admin=True, contributor__node=self.machineable.target):
```

Comment:
Minor: Is `contributor__node=...` necessary? Won't `self.machineable.target` automatically do that filter?

I'm not sure if it makes any difference--just curious, really.